src/_common/community/card/AppCommunityCard.vue

```diff
@@ -0,0 +1,44 @@
+<script lang="ts" setup>
+import { PropType, toRefs } from 'vue';
+import { trackGotoCommunity } from '../../analytics/analytics.service';
+import AppCommunityCardBase from '../card-base/AppCommunityCardBase.vue';
+import { CommunityModel } from '../community.model';
+import AppCommunityThumbnailImg from '../thumbnail/AppCommunityThumbnailImg.vue';
+
+const props = defineProps({
+	community: {
+		type: Object as PropType<CommunityModel>,
+		required: true,
+	},
+	elevate: { type: Boolean },
```

Comment:
we always put these on multiple lines like so:
```
elevate: {
    type: Boolean,
},
```

it just makes it more consistent